### ISSUE TITLE
Identify all wiki pages that can be safely merged or delted [WIP list]

### ISSUE THREAD

**Issue by silverhook** [fossology/fossology#1364]
This is a meta-issue collecting and keeping track of the merging of the two wikis into one.

List (WIP) of wiki pages that could be merged and/or removed:

- [ ] Update Debian packaging documentation #299
- [ ] Merge duplicate wiki pages #815
- [ ] …

**Comment by mcjaeger** [comment]
I think #299 is actually important

**Comment by silverhook** [comment]
> I think #299 is actually important

Yeah, but we probably should migrate and update the info there.

**Comment by GMishx** [comment]
Adding to the list: https://github.com/fossology/fossology/wiki/List-of-VM-Systems

**Comment by silverhook** [comment]
@GMishx, I suspect both are simply ripe for removal? Or what did you have in mind?

**Comment by GMishx** [comment]
Yes @silverhook, they are no longer relevant so can be removed simply.

**Comment by silverhook** [comment]
@GMishx, thanks. I removed them now.

**Comment by Aman-Codes** [comment]
Hi @silverhook @GMishx 
The wiki page https://github.com/fossology/fossology/wiki/Access-Control and https://github.com/fossology/fossology/wiki/Permissions-and-Groups are exactly identical so one of them can be deleted.

**Comment by silverhook** [comment]
> Hi @silverhook @GMishx
> The wiki page https://github.com/fossology/fossology/wiki/Access-Control and https://github.com/fossology/fossology/wiki/Permissions-and-Groups are exactly identical so one of them can be deleted.

Good catch, @Aman-Codes.

I just changed the content of Premissions-And-Groups to point to Access-Control, in case anyone still looks at that page.

**Comment by mayank-pathakk** [comment]
@silverhook @GMishx I think we should also merge 
https://github.com/fossology/fossology/wiki/FOSSology-REST-API
and 
https://github.com/fossology/fossology/wiki/REST-API
